### PR TITLE
Pseudo main

### PR DESCRIPTION
**Pull request by ZeniaHansen**
## Description

Merged Node Template into Exercise repository

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Feature that assists further development (non-breaking change for the developer)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?
Monkey test

### Checklist:

- [x] You have added a clear title and description for the PR
- [x] The CI pass
- [x] There are no merge conflicts
- [x] The code satisfies the [code standards](https://docs.google.com/document/d/1R9n9MeuRwz3WSrnWZUjrbmGeosTlu1Xz8f5F7gu2AJE/edit#heading=h.e9xl4xfoldow) 
- [ ] I have added docstrings for my code, where needed
- [x] Code implements intended functionality
- [x] All relevant tests pass (Black, White, Integration, Monkey, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-Microservice to handle exercises
+[![Continous Integration](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_integration.yml)
```
Comment (Brandhoej) [inline README.md]: ``OnlineTrainingPlatform/nodeTemplate`` should be ``OnlineTrainingPlatform/exercises``

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+[![Continous Delivery](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_delivery.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_delivery.yml)
```
Comment (Brandhoej) [inline README.md]: ``OnlineTrainingPlatform/nodeTemplate`` should be ``OnlineTrainingPlatform/exercises``

**Review by Brandhoej — COMMENTED** [review]
Badges should reference the correct repository

File: README.md (hunk 1 above)
Comment (ZeniaHansen) [inline README.md]:
```suggestion
[![Continous Integration](https://github.com/OnlineTrainingPlatform/Exercises/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/Exercises/actions/workflows/continous_integration.yml)
```

File: README.md [hunk 3]
```diff
@@ -1 +1,49 @@
-Microservice to handle exercises
+[![Continous Integration](https://github.com/OnlineTrainingPlatform/Exercises/actions/workflows/continous_integration.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/Exercises/actions/workflows/continous_integration.yml)
+[![Continous Delivery](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_delivery.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/nodeTemplate/actions/workflows/continous_delivery.yml)
```
Comment (ZeniaHansen) [inline README.md]:
```suggestion
[![Continous Delivery](https://github.com/OnlineTrainingPlatform/Exercises/actions/workflows/continous_delivery.yml/badge.svg)](https://github.com/OnlineTrainingPlatform/Exercises/actions/workflows/continous_delivery.yml)
```

**Review by Brandhoej — APPROVED** [review]
Looks good!

**Review by Dieuer — APPROVED** [review]
Good job

**Pull request merged**